maestro-cli/src/main/java/maestro/cli/report/JUnitTestSuiteReporter.kt

```diff
@@ -40,9 +40,9 @@ class JUnitTestSuiteReporter(
                                 testCases = suite.flows
                                     .map { flow ->
                                         TestCase(
-                                            id = flow.fileName ?: flow.name,
+                                            id = flow.name,
                                             name = flow.name,
-                                            classname = flow.fileName ?: flow.name,
+                                            classname = flow.name,
```

Comment:
Cloud runs and local runs produce different names. This is done for consistency of output.
This will indeed change naming behaviour, depending on user configuration.